build.gradle

```diff
@@ -132,7 +132,7 @@ dependencies {
     // transitive dependencies.
     constraints {
         implementation('org.scala-lang:scala-library:2.13.13')
-        implementation('org.json:json:20240205')
+        implementation('org.json:json:20240303')
```

Comment:
I'm not sure what to make of this. The change looks fine but I don't this constraint is needed anymore. Actually I think all of these constraints can be removed, although removing the one on `io.grpc` would also require updating the google cloud BOM version.

Reply:
I think that all sounds good -- and I think the google cloud BOM upgrade is in a grouped Dependabot PR now anyway, so it should be no problem to pull it out and update separately.

Reply:
Makes sense to me to just remove the constraint.

Reply:
Great, see https://github.com/DataBiosphere/terra-common-lib/pull/176. If that merges I can close this PR.

Reply:
I love that Dependabot cleans up after itself :)